### ISSUE TITLE
The result is not that accurate

### ISSUE THREAD

**Issue by ILG2021** [kyaw-yethu/burmese-G2P#1]
May be our train data is tiny, the g2p result is not that accurate. I use ver 2, I listen to the audio and compare the results. ChatGPT is much more accurate. Can we improve it?

**Comment by kyaw-yethu** [comment]
I haven't listened to the audio generated from IPA phonemes, but I suspect inaccuracies stem from the model's inability to natural-soundingly integrate the particles and post-positional markers in the sentence based on the characters surrounding them. For this sentence-level conversion can be helpful. Also, CRF I used might be a fairly basic model for this task compared with deep learning models with attention. 

Also, I doubt I can actively contribute to this repo any longer. If you somehow plan to contribute to it, I'll appreciate that.

**Comment by ILG2021** [comment]
Thank you very much. I have noticed that many syllables have multiple pronounce. I am planning a byt5 model for this.
Can you share more train data than myG2P?

**Comment by kyaw-yethu** [comment]
Unfortunately, I don't know any other data source than the parent repo of this one. Before we're certain that training data is the limiting factor, it'd be insightful to see the performance with byt5 using the current data. With such a pre-trained model, the current size of training data might be enough since it'd be just fine-tuning. Actually, we can also combine the test set with the training set and use it entirely for training to get additional 5000 lines of training data. But we'd be  only able to evaluate qualitatively by the generated audio from the predicted phonemes.
